lib/kontena/machine/digital_ocean/cloudinit.yml

```diff
@@ -48,6 +52,12 @@ coreos:
         Name=weave*
```

Comment:
If we're overriding the `zz-default.network` to not match the weave bridge, then I think we don't need this `10-weave.network` anymore. It probably hurts more than it helps, if it triggers netlink races with `weave attach`.